.circleci/config.yml

```diff
@@ -737,6 +737,7 @@ jobs:
             pulumi config set --path unchained:common.eks.traefik.whitelist[20] $AWS_TAXISTAKE_NAT_ADDRESS_0
             pulumi config set --path unchained:common.eks.traefik.whitelist[21] $AWS_TAXISTAKE_NAT_ADDRESS_1
             pulumi config set --path unchained:common.eks.traefik.whitelist[22] $AWS_TAXISTAKE_NAT_ADDRESS_2
+            pulumi config set --path unchained:common.eks.volumeSize 100
```

Comment:
Looks like I'll have a bit of Pulumi docs to read haha but this is fairly straightforward, noice